src/release.js

```diff
@@ -551,6 +551,44 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
             }
         },
 
+
+        /**
+         * Merge release branch into releasing branch and ask user to resolve conflicts manually if any
+         */
+        async mergeWithReleaseBranch() {
+            log.doing(`Merging '${releaseBranch}' into '${data.releasingBranch}'.`);
+
+            try {
+                // checkout master
+                await gitClient.checkout(releaseBranch);
+
+                // pull master
+                await gitClient.pull(releaseBranch);
+
+                // checkout releasingBranch
+                await gitClient.checkout(`${branchPrefix}-${data.version}`);
+
+                // merge release branch into releasingBranch
+                await gitClient.merge([releaseBranch]);
+
+                log.done(`'${releaseBranch}' merged into '${branchPrefix}-${data.version}'.`);
+            } catch (err) {
+                log.warn('Please resolve the conflicts and complete the merge manually (including making the merge commit).');
```

Comment:
It would be nice if we could check the properties of the `err` object. Because in some cases, it could be not the merge step, but one of the previous commands which throws an error.

Reply:
done